app/[id]/goods/index.tsx

```
@@ -0,0 +1,181 @@
+    })
+
+    return (
+        <MenuProvider>
+            <View style={styles.container}>
+                {showMenu && (
+                    <>
+                        <Pressable
+                            className="absolute top-0 left-0 right-0 bottom-0 bg-black/20 z-40"
+                            onPress={() => setShowMenu(false)}
+                        />
+                        <Menu onBackdropPress={() => setShowMenu(false)}>
```

ESLint

[error] 'Menu' is not defined.

(react/jsx-no-undef)